vendor/src/github.com/docker/libcontainer/nsinit/pause.go

```diff
@@ -39,7 +39,7 @@ func toggle(state cgroups.FreezerState) error {
 		return err
 	}
 
-	if systemd.UseSystemd() {
+	if container.Cgroups.Driver == "systemd" && systemd.UseSystemd() {
```

Comment:
If you look at the patch in libcontainer that's linked from this PR you
will see the circular dependency problem

On Wednesday, January 28, 2015, Kevin Ran <notifications@github.com> wrote:

> In vendor/src/github.com/docker/libcontainer/nsinit/pause.go
> <https://github.com/docker/docker/pull/10427#discussion_r23744403>:
>
> > @@ -39,7 +39,7 @@ func toggle(state cgroups.FreezerState) error {
> >  		return err
> >  	}
> >
> > -	if systemd.UseSystemd() {
> > +	if container.Cgroups.Driver == "systemd" && systemd.UseSystemd() {
>
> much container.Cgroups.Driver == "systemd". Is there a reason why it
> shouldn't just be container.Cgroups.IsSystemd()?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/10427/files#r23744403>.
>